src/build/openapi.ts

```diff
@@ -70,6 +70,53 @@ export async function generateOpenApi(project: Project) {
 				},
 			};
 		}
+
+		for (const route of mod.routes.values()) {
+			let methods: string[];
+			if (route.config.method) {
+				methods = [route.config.method.toLowerCase()];
+			} else {
+				methods = ["get", "post", "put", "patch", "delete"];
+			}
+
+			const methodConfig: Record<string, unknown> = {
+				description: `Call ${mod.name}.${route.name} route.`,
+				tags: ["Backend"],
+				operationId: `call_${mod.name}_${route.name}`,
```

Comment:
won't this potentially have a naming conflict with a script?

Reply:
yes, will fix